packages/menu-button/package.json

```diff
@@ -5,6 +5,7 @@
   "author": "Ryan Florence <@ryanflorence>",
   "license": "MIT",
   "main": "index.js",
+  "types": "index.d.ts",
```

Comment:
I think this line is not required because typescript will, by default, detect an index.d.ts file at the root of a package

Reply:
You are correct that it is not needed, however the TypeScript documentation still recommends to be explicit:

> Also note that if your main declaration file is named index.d.ts and lives at the root of the package (next to index.js) you do not need to mark the "types" property, though it is advisable to do so

From: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html